Radio_continuum_validation.py

```diff
@@ -0,0 +1,213 @@
+                            each file not found in given path) [default: NVSS_config.txt,SUMSS_config.txt,GLEAM_config.txt,TGSS_config.txt].
+  -N --noise=<map>          Use this fits image of the local rms. Default is to run BANE [default: None].
+  -F --filter=<config>      A config file for filtering the sources in the input fits file [default: None].
+  -R --snr=<ratio>          The signal-to-noise ratio cut to apply to the input catalogue and the source counts (doesn't affect source finding) [default: 5.0].
+  -v --verbose              Verbose output [default: False].
+  -f --refind               Force source finding step when catalogue already exists (sets redo to True) [default: False].
+  -r --redo                 Force every step again (except source finding), even when catalogues already exist [default: False].
+  -p --peak-flux            Use the peak flux rather than the integrated flux of the input image (not used when -A used) [default: False].
+  -w --write                Write intermediate files generated during processing (e.g. cross-matched and pre-filtered catalogues, etc).
+                            This will save having to reprocess the cross-matches, etc when executing the script again. [default: False].
+  -x --no-write             Don't write any files except the html report and any files output from BANE and Aegean. [default: False].
+  -m --SEDs=<models>        A comma-separated list of SED models to fit to the radio spectra ('pow','SSA','FFA','curve',etc) [default: None].
+  -e --SEDfig=<extn>        Write figures for each SED model with this file extension (may significantly slow down script) [default: None].
+  -t --telescope=<name>     Unique name of the telescope or survey to give to the main catalogue (not used when -A used). [default: ASKAP].
+  -d --main-dir=<path>      The absolute path to the main directory where this script and other required files are located [default: $ACES/UserScripts/col52r].
+  -n --ncores=<num>         The number of cores (per node) to use when running BANE and Aegean (using >=20 cores may result in memory error) [default: 8].
+  -b --nbins=<num>          The number of bins to use when performing source counts [default: 50].
+  -s --source=<src>         The format for writing plots (e.g. screen, html, eps, pdf, png, etc) [default: html].
+  -a --aegean=<params>      A single string with any extra paramters to pass into Aegean (except cores, noise, background, and table) [default: --floodclip=3].
+  -c --correct=<level>      Correct the input fits image, write to 'name_corrected.fits' and use this to run through 2nd iteration of validation.
+                            Fits image is corrected according to input level (0: none, 1: positions, 2: positions + fluxes) [default: 0]."""
+
+import os
+import sys
+import glob
+import warnings
+import matplotlib as mpl
+from functions import *
```

Comment:
Star imports are considered bad practice. 
`from functions import *` 
It's not clear where this is being used in the code.

I suggest import implicitly:
`from functions import a_function b_function` 

or import the name space
'import functions'

Reply:
From flake8:
```
catalogue.py:682:34: F405 'axis_lim' may be undefined, or defined from star imports: functions
catalogue.py:682:53: F405 'axis_lim' may be undefined, or defined from star imports: functions
catalogue.py:683:36: F405 'axis_lim' may be undefined, or defined from star imports: functions
catalogue.py:683:56: F405 'axis_lim' may be undefined, or defined from star imports: functions
catalogue.py:1149:36: F405 'flux_at_freq' may be undefined, or defined from star imports: functions
catalogue.py:1179:68: F405 'two_freq_power_law' may be undefined, or defined from star imports: functions
catalogue.py:1297:66: F405 'SED' may be undefined, or defined from star imports: functions
catalogue.py:1366:22: F405 'config2dic' may be undefined, or defined from star imports: functions
```

Reply:
Done.